MDANSE/Src/MDANSE/Framework/Configurators/AtomSelectionConfigurator.py

```diff
@@ -132,14 +134,11 @@ def get_information(self) -> str:
 
         return "\n".join(info) + "\n"
 
-    def get_selector(self) -> Selector:
+    def get_selector(self) -> "ReusableSelection":
```

Comment:
Am I wrong in thinking this is imported?
```suggestion
    def get_selector(self) -> ReusableSelection:
```